src/rgw/rgw_admin.cc

```diff
@@ -309,15 +309,18 @@ void usage()
   cout << "  mfa remove                 delete MFA TOTP token\n";
   cout << "  mfa check                  check MFA TOTP token\n";
   cout << "  mfa resync                 re-sync MFA TOTP token\n";
-  cout << "  topic list                 list bucket notifications topics\n";
-  cout << "  topic get                  get a bucket notifications topic\n";
-  cout << "  topic rm                   remove a bucket notifications topic\n";
+  cout << "  topic list                 list notifications topics\n";
+  cout << "  topic get                  get a notifications topic\n";
+  cout << "  topic rm                   remove a notifications topic\n";
   cout << "  script put                 upload a lua script to a context\n";
   cout << "  script get                 get the lua script of a context\n";
   cout << "  script rm                  remove the lua scripts of a context\n";
   cout << "  script-package add         add a lua package to the scripts allowlist\n";
   cout << "  script-package rm          remove a lua package from the scripts allowlist\n";
   cout << "  script-package list        get the lua packages allowlist\n";
+  cout << "  notification list          list bucket notifications\n";
```

Comment:
```suggestion
  cout << "  notification list          list bucket notifications configuration\n";
```

Reply:
otherwise, it is going to confuse users that they will be getting the notifications themselves

Reply:
Got it!